converters/fromZigbee.js

```diff
@@ -5686,6 +5691,55 @@ const converters = {
             return payload;
         },
     },
+    silvercrest_smart_led_string: {
+        cluster: 'manuSpecificTuya',
+        type: ['commandGetData', 'commandSetDataResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const dp = msg.data.dp;
+            const value = tuyaGetDataValue(msg.data.datatype, msg.data.data);
+
+            if (dp === common.TuyaDataPoints.silvercrestChangeMode) {
+                meta.state.mode = value;
+            } else if (dp === common.TuyaDataPoints.silvercrestSetBrightness) {
+                meta.state.brightness = value;
+            } else if (dp === common.TuyaDataPoints.silvercrestSetColor) {
+                const hString = value.substring(0, 4);
+                const sString = value.substring(4, 8);
+                const bString = value.substring(8, 12);
+
+                const h = parseInt(hString, 16);
+                const s = parseInt(sString, 16);
+                const b = parseInt(bString, 16);
+
+                if (!meta.state.color) {
+                    meta.state.color = {};
+                }
+                meta.state.color.b = (b / 1000) * 255;
+                meta.state.color.h = h;
+                meta.state.color.s = s / 10;
+            } else if (dp === common.TuyaDataPoints.silvercrestSetScene) {
+                meta.state.scene = {
```

Comment:
- Scene -> effect
- Can we return the effect name (string) instead here? 
-  I don't understand the purpose of https://github.com/Koenkk/zigbee-herdsman-converters/pull/1799/files#diff-b676ab0df4d49b8572967cb4c92a2dc6264091131f6b1828ca13d0f9656e367aR5731

Reply:
Scene / effect: now it's according to the Lidl Home Android app.
Effect as string: Sure
Purpose of the loop: it could return one color, or it could return 6 colors. That string is variable. I use that loop to cut the response in pieces to set the colors.